lib/Test2/Harness/Renderer/JUnit.pm

```diff
@@ -107,7 +108,7 @@ sub render_event {
         $test->{'testsuite'}->{'timestamp'} = _timestamp( $test->{'start'} );
         $test->{'testsuite'}->{'id'} = $self->{'suite_id'}++;
 
-        push @{ $test->{'testcase'} }, $self->xml->testcase( { 'name' => "Tear down.", 'time' => $stamp - $test->{'last_job_start'} }, "" );
+        push @{ $test->{'testcase'} }, $self->xml->testcase( { 'name' => "Tear down.", 'classname' => $test->{'file'}, 'time' => $stamp - $test->{'last_job_start'} }, "" );
```

Comment:
Adding classname needs to be optional. it confuses the heck out of jenkins if I recall correctly which is why I took it out.